operator/gitops/argocd/pipeline-service/tekton-results/api-s3-config.yaml

```diff
@@ -43,5 +43,12 @@ spec:
                 secretKeyRef:
                   key: endpoint
                   name: tekton-results-s3
+          resources:
+            requests:
+              cpu: 5m
+              memory: 32Mi
+            limits:
+              cpu: 100m
+              memory: 128Mi
```

Comment:
On prd-rh01, I see a peak at 224Mi during normal operations. Can we bump that value to 256.